### ISSUE TITLE
Lutris fails with `No such file or directory` when installing something

### ISSUE THREAD

**Issue by lovewestlundgotby** [NixOS/nixpkgs#137367]
### Describe the bug
I am unable to install anything in Lutris, where it fails with `No such file or directory` on at least

 - `~/.local/share/lutris/runners/wine/lutris-fshack-6.14-3-x86_64/bin/wine`
 - `~/.local/share/lutris/runners/wine/lutris-fshack-6.14-3-x86_64/bin/wineserver`
 - `~/.local/share/lutris/runtime/p7zip/7z`

even though those executables are present on my system.

My research on this tells me that the error produced is a bit misleading, as the root cause is that the binary is trying to use for example `/lib/ld-linux.so.2` and that obviously does not exist on NixOS.

### Steps To Reproduce
1. install `lutris-unwrapped`
2. run `lutris`
3. try to install anything.

### Expected behavior
The lutris installer to finish successfully, or at least not error on what is described here.

### Screenshots
N/A

### Additional context
I am aware that [patchelf](https://github.com/NixOS/patchelf) can be used to fix this kind of thing, the question is if every user is expected/have to do this manually or if it could be done when installing the `lutris-unwrapped` package. Or did I miss something? I am rather new to NixOS after all.

### Notify maintainers
@Chiiruno 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.62, NixOS, 21.11 (Porcupine)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.4pre20210802_47e96bb`
 - nixpkgs: `/nix/store/yfjhaanny02sk592mnw2ddr87qalki49-source`
```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
  - lutris-unwrapped
# a list of nixos modules affected by the problem
module:
```

**Comment by kira-bruneau** [comment]
@lovewestlundgotby You should be using `lutris` instead of `lutris-unwrapped`, `lutris` wraps `lutris-unwrapped` in an fhs emulated environment so you can run those executables.

**Comment by lovewestlundgotby** [comment]
That makes a lot of sense, and probably solves my issue even though I have not tested it yet. 

To answer your question, I am not sure why I ended up with `lutris-unwrapped`. I think that I tested both and noticed that I got a `lutris` executable from both, and after only skimming through the nix-files I went with `lutris-unwrapped` because it was mentioned in the latest commit to `lutris/default.nix`. Guess I should have read a bit more in detail in `lutris/fhsenv.nix` where I now see the wrapping being done. 

Anyway, I guess that this also serves its purpose to tell other newcomers which one to use!
